doc/guides/index.rst

```diff
@@ -25,3 +25,4 @@ User and Developer Guides
    test/index
    tracing/index
    west/index
+   modules.rst
```

Comment:
It looks like these are in alphabetical order by document title (not file name); putting this at the end breaks that.